lib/rspec/graphql_matchers/have_a_return_field.rb

```diff
@@ -0,0 +1,16 @@
+module RSpec
+  module GraphqlMatchers
+    class HaveAReturnField < HaveAField
+      def initialize(expected_field_name)
+        super
+        @fields = :return_fields
+      end
+
+      def matches?(graph_object)
```

Comment:
It seems this method is the same and can be inherited directly from HaveAField now (same for HaveAnInputField). Am I missing something?